tslib/src/functions/check-receiver/index.ts

```diff
@@ -1,8 +1,14 @@
+/* eslint-disable import/first */
+// intentionally named incorrectly to test app insights in azure.  will be corrected
+const functionName = 'the-check-receiver'

```

Comment:
```suggestion
const functionName = 'check-receiver'

```

Reply:
this is intentional, as per comment on the line above.

it will be reverted in a future PR, which i have a ticket for.